mmcls/models/losses/cross_entropy_loss.py

```diff
@@ -75,19 +107,28 @@ class CrossEntropyLoss(nn.Module):
     Args:
         use_sigmoid (bool): Whether the prediction uses sigmoid
             of softmax. Defaults to False.
+        use_soft (bool): Whether to use the soft version of CrossEntropyLoss.
+            Defaults to False.
         reduction (str): The method used to reduce the loss.
             Options are "none", "mean" and "sum". Defaults to 'mean'.
         loss_weight (float):  Weight of the loss. Defaults to 1.0.
     """
 
-    def __init__(self, use_sigmoid=False, reduction='mean', loss_weight=1.0):
+    def __init__(self,
+                 use_sigmoid=False,
+                 use_soft=False,
+                 reduction='mean',
+                 loss_weight=1.0):
         super(CrossEntropyLoss, self).__init__()
         self.use_sigmoid = use_sigmoid
+        self.use_soft = use_soft
         self.reduction = reduction
         self.loss_weight = loss_weight
 
```

Comment:
use_soft and use_sigmoid could not be set simultaneously, so we'd better add some assertions here.